appcues/src/main/java/com/appcues/ui/primitive/EmbedHtmlPrimitive.kt

```diff
@@ -63,15 +99,13 @@ internal class EmbedChromeClient(private val container: ViewGroup) : Accompanist
         customView = view
         customCallback = callback
         container.addView(view)
-        container.visibility = View.VISIBLE
     }
 
     override fun onHideCustomView() {
         if (customView == null) {
             return
         }
         customView?.visibility = View.GONE
-        container.visibility = View.GONE
```

Comment:
this is the one bug fix - this is currently breaking the return from fullscreen video in v3.0 - setting the container view GONE - this is just the root view of the decorView now (not a separate overlay view) so we cannot set it GONE.